tests/foreman/api/test_docker.py

```diff
@@ -272,13 +221,11 @@ def test_sync_docker_repo(self):
         @Feature: Docker
 
         """
-        prod_id = entities.Product(
-            organization=self.org_id
-        ).create_json()['id']
-        repo_id = _create_repository(prod_id)['id']
-
-        entities.Repository(id=repo_id).sync()
-        attrs = entities.Repository(id=repo_id).read_json()
+        repo = _create_repository(
+            entities.Product(organization=self.org).create()
+        )
+        repo.sync()
+        attrs = repo.read_json()
```

Comment:
Yes, this can be improved. I/we should start filing issues for each of these missing attributes.